docs/index.rst

```diff
@@ -272,7 +272,7 @@ In this demo, KSQL is authenticated and authorized to connect to the secured Kaf
 
    .. sourcecode:: bash
 
-        docker-compose exec ksql-cli bash -c 'ksql -u ksqlUser -p ksqlUser http://ksql-server:8088'
+        docker-compose exec ksql-cli bash -c 'ksql -u ksqlUser -p ksqlUser http://ksqldb-server:8088'
```

Comment:
Is the plan to keep the CLI container ksql-cli (instead of ksqldb-cli)?

Reply:
Good catch, renaming now